packages/xstate-react/test/useService.test.tsx

```diff
@@ -173,4 +182,59 @@ describe('useService hook', () => {
     fireEvent.click(incButton);
     expect(countEl.textContent).toBe('1');
   });
+
+  it('initial invoked actor should be immediately available', (done) => {
+    const childMachine = createMachine({
+      id: 'childMachine',
+      initial: 'active',
+      states: {
+        active: {
+          on: {
+            FINISH: { actions: sendParent('FINISH') }
+          }
+        }
+      }
+    });
+    const machine = createMachine({
+      initial: 'active',
+      invoke: {
+        id: 'child',
+        src: childMachine
+      },
+      states: {
+        active: {
+          on: { FINISH: 'success' }
+        },
+        success: {}
+      }
+    });
+
+    const ChildTest: React.FC<{ actor: Actor<any> }> = ({ actor }) => {
+      const [state, send] = useActor(actor);
+
+      expect(state.value).toEqual('active');
+
+      React.useEffect(() => {
+        send({ type: 'FINISH' });
+      }, []);
+
+      return null;
+    };
+
+    const Test = () => {
+      const [state] = useMachine(machine);
+
+      if (state.matches('success')) {
+        done();
+      }
+
+      return <ChildTest actor={state.children.child} />;
+    };
+
+    render(
+      <React.StrictMode>
```

Comment:
I don't think `StrictMode` is that relevant for this test - it doesn't matter (for this test) if you wrap your root component with it or not.

Reply:
I actually think that `StrictMode` should be used for _all_ the tests if we want to catch unforeseen issues with CM.

Reply:
fair point